src/storage/txn/scheduler.rs

```diff
@@ -874,9 +937,19 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
                     if self.inner.flow_controller.enabled()
                         && !self.inner.flow_controller.is_unlimited()
                     {
+                        let mut deadline_fut = GLOBAL_TIMER_HANDLE
+                            .delay(deadline.to_std_instant())
+                            .compat()
+                            .fuse();
```

Comment:
Just curious, why need `fuse` here?

Reply:
It's a requirement of the `select!` macro.
https://docs.rs/futures/0.3.16/futures/macro.select.html